### PR TITLE
Improving accessibility on the new interface at /v2

### PR DESCRIPTION
**Pull request by waridrox**
Fixed contrast issues and added relevant aria-labels and alt tags following WCAG guidelines 

Following is the result after the implementation 

  
accessibility

- The remaining colour contrast issues at `Capture` tab for instance are due to overlapping elements (like the flot js graph) which are non-identifiable by the [_Axe Web Accessibility tool_](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd) hence shouldn't be a problem. 
- Rest of the issues direct associating captions on the `vid` element, which are irrelevant, since here there is no need for captions

Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- `rake test`
* [ ] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [ ] pull request are descriptively named
* [ ] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Review by jywarren — APPROVED** [review]

**Comment by jywarren** [comment]
Very nicely done! Much appreciated!!!

File: app/views/capture/index2.html.erb [hunk 1]
```diff
@@ -1,4 +1,5 @@
-<script src="https://cdn.temasys.io/adapterjs/0.15.x/adapter.min.js"></script>
+<html lang="en" role="main">
+<%= javascript_include_tag "adapterjs/publish/adapter.min.js" %>
```
Comment (jywarren) [inline app/views/capture/index2.html.erb:2]: Ah, i missed this. I think we need to add this to package.json.

Comment (jywarren) [reply app/views/capture/index2.html.erb:2]: https://www.npmjs.com/package/adapterjs

Comment (waridrox) [reply app/views/capture/index2.html.erb:2]: Oh my bad, I think I missed that 😅

Comment (waridrox) [reply app/views/capture/index2.html.erb:2]: Will need to update yarn.lock too...that way the yarn.lock file gets referenced in actions...

Comment (jywarren) [reply app/views/capture/index2.html.erb:2]: Whew, this published to the live site. Getting a fix in now!

OK: https://github.com/publiclab/spectral-workbench/commit/1091951a46bf2a5f9ee9abc1b3e394936bdf976a

Let's test this out in stable or at least GitPod in the future! But seems all good now... will deploy shortly. Thanks!

Comment (waridrox) [reply app/views/capture/index2.html.erb:2]: Oh I made another one at #704 😅

**Pull request merged**